conda_forge_tick/update_sources.py

```diff
@@ -590,6 +590,35 @@ def get_url(self, meta_yaml) -> Optional[str]:
         return super().get_url(meta_yaml)
 
 
+class GitTags(AbstractSource):
+    name = "GitTags"
+
+    def get_url(self, meta_yaml):
+        return meta_yaml.get("git_url")
```

Comment:
This key will not be available at the top level in the node attributes. The name of the variable is confusing here (not your fault) and the code paths are very confusing for parsing (not your fault). 

The url and the hash are added as special cases [here](https://github.com/regro/cf-scripts/blob/main/conda_forge_tick/feedstock_parser.py#L530).

Instead, you'll need to look directly in the meta_yaml key at the source section.

Reply:
Added in [c4266d5](https://github.com/regro/cf-scripts/pull/4811/commits/c4266d569a563a355a3e6d6e4637609d6e8a221e), thanks!